cli/testdata/coder_server_--help.golden

```diff
@@ -54,10 +54,29 @@ OPTIONS:
           Type of auth to use when connecting to postgres.
 
       --postgres-url string, $CODER_PG_CONNECTION_URL
-          URL of a PostgreSQL database. If empty, PostgreSQL binaries will be
-          downloaded from Maven (https://repo1.maven.org/maven2) and store all
-          data in the config root. Access the built-in database with "coder
-          server postgres-builtin-url".
+          URL of a PostgreSQL database. If empty and credentials are not
+          provided via other flags, PostgreSQL binaries will be downloaded from
+          Maven (https://repo1.maven.org/maven2) and store all data in the
+          config root. Access the built-in database with "coder server
+          postgres-builtin-url".
```

Comment:
We should also document that this can't be used together with the other options.